ext/HealthBaseOMOPCDMExt.jl

```diff
@@ -0,0 +1,108 @@
+module HealthBaseOMOPCDMExt
+
+using HealthBase
+using DataFrames
+using OMOPCommonDataModel
+using Serialization
+using InlineStrings
+using Dates
+
+const OMOPCDM_VERSIONS = Dict{Any, Any}()
+
+# Mapping OMOP CDM datatypes to Julia types
+const datatype_map = Dict(
+    "integer" => Int64, "Integer" => Int64, "bigint" => Int64,
+    "float" => Float64,
+    "date" => Date, "datetime" => DateTime,
+    "varchar(1)" => String, "varchar(2)" => String, "varchar(3)" => String,
+    "varchar(9)" => String, "varchar(10)" => String, "varchar(20)" => String,
+    "varchar(25)" => String, "varchar(50)" => String, "varchar(80)" => String,
+    "varchar(250)" => String, "varchar(255)" => String, "varchar(1000)" => String,
+    "varchar(2000)" => String, "varchar(MAX)" => String
+)
+
+function __init__()
+    @info "OMOP CDM extension for HealthBase has been loaded!"
+    merge!(OMOPCDM_VERSIONS, deserialize(joinpath(@__DIR__, "..", "assets", "version_info")))
```

Comment:
Can't you just declare this constant outside the initialization function? Also, why are you merging? Can you do `const OMOPCDM_VERSIONS = deserialize(......)`? 

Also, can you leave a note comment (i.e. `# NOTE: `) for me so I do not forget to replace this constant line with dependencies from OMOPCommonDataModel.jl?